addons/mail/demo/discuss_channel_demo.xml

```diff
@@ -233,5 +233,39 @@
             <field name="seen_message_id" ref="mail.troubleshooting_message_2"/>
             <field name="new_message_separator" eval="ref('mail.troubleshooting_message_2') + 1"/>
         </record>
+        <record model="mail.message" id="mail.channel_public_message_0">
```

Comment:
Channel demo stuff would be best on a separate `discuss` sub-folder in the ever lasting preparation for a split.

Also I would suggest 1 file per channel (similar to what is done in livechat) it makes things much easier to find/maintaing.